src/service/viewport-impl.js

```diff
@@ -342,6 +348,18 @@ export class Viewport {
   }
 
   /**
+   * Returns the first viewport rect.
+   * @return {!../layout-rect.LayoutRectDef}}
+   */
+  getFirstViewportRect() {
```

Comment:
first viewport is always the same: (0, 0, size.width, size.height). We don't really need this property and can calculate where it's needed.